### PR TITLE
Prepare release of SDK 1.5.3

### PR DESCRIPTION
**Pull request by bertrand-lorentz**

### CODE REVIEW

**Comment by bertrand-lorentz** [comment]
View templates were all successfully compiled manually. This confirms their syntax is valid and they do not reference any node or field that does not exist in this version.

File: translations/auxiliary_ga.xml
Comment (rousso) [inline translations/auxiliary_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/business-term_ga.xml
Comment (rousso) [inline translations/business-term_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/code_ga.xml
Comment (rousso) [inline translations/code_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/view_ga.xml
Comment (rousso) [inline translations/view_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/rule_ga.xml
Comment (rousso) [inline translations/rule_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/notice_ga.xml
Comment (rousso) [inline translations/notice_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/group_ga.xml
Comment (rousso) [inline translations/group_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/field_ga.xml
Comment (rousso) [inline translations/field_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

File: translations/codelist_ga.xml
Comment (rousso) [inline translations/codelist_ga.xml:1]: Same issue with review of 1.6.1. The filename seems wrong.

**Review by rousso — CHANGES_REQUESTED** [review]
Same issues with review of 1.6.1. The indicated filenames seem wrong.

**Pull request merged**